Automattic/wp-calypso
# E2E: Fix E2E tests for fse-template-events

### Proposed Changes

This PR fixes broken tests for editor-tracking > fse-template-events. Many issues stem from wrong selectors. It seems likely these selectors (part of WordPress core code) have simply changed since our E2E tests were originally written. 

### Testing Instructions

1) Set up your testing environment following the instructions here: https://github.com/Automattic/wp-calypso/tree/trunk/test/e2e

2) One set up, run the relevant tests with `yarn jest specs/editor-tracking/editor-tracking__fse-template-events`. You should see all tests (including 1 test suite and 19 tests) pass.
